components/ai_chat/resources/page/components/main/index.tsx

```diff
@@ -57,8 +57,22 @@ function Main() {
     siteInfo === null && // SiteInfo request has finished and this is a standalone conversation
     !context.isPremiumUser
 
+  const isLastTurnBraveSearchSERPSummary = () => {
+    const lastTurn = context.conversationHistory.at(-1);
```

Comment:
it's confusing that some of these lines end in semicolons and some don't; i would expect the linter to enforce that all of them don't

Reply:
ah, didn't realize I added ; here, I'm going to remove them all.

Reply:
fixed in https://github.com/brave/brave-core/pull/25257/commits/6d742eb1b19c625657a822ca42b872122de5f369